main.py

```diff
@@ -0,0 +1,515 @@
+        self.image = pygame.image.load(CURR_DIR + '/images/arrow2.png')
+        self.rect = self.image.get_rect()
+        self.rect.left = x
+        self.rect.top = y
+
+    def update(self):
+        if self.active:
+            if self.rect.top <= 0:                                      # disappears when hits top of screen                      
+                self.active = False
+                self.kill()
+            else:
+                self.rect.top -= 10
+        else:
+            self.kill()
+
+class Ball(pygame.sprite.Sprite):
+    """ Bubbles that character needs to pop """
+    def __init__(self, size, speed, x = 0, y = 0):
+        pygame.sprite.Sprite.__init__(self)
+        self.size = size
+        self.speed = speed
+        self.pic = random.choice(pictures)                              # randomly chooses between given pictures of Olin faces
+        self.image = pygame.image.load(self.pic)
+        self.image = pygame.transform.scale(self.image,(size*10,size*10))
+        self.rect = self.image.get_rect()
+        self.rect.left = x
+        self.rect.top = y
+
+    def update(self):
+        self.rect = self.rect.move(self.speed)
+
+        if self.rect.left < 0 or self.rect.right > screen_width:        # bubbles bounce off of left/right/top/bottom walls
+            self.speed[0] = -self.speed[0]
+        if self.rect.top < 100 or self.rect.bottom > screen_height:
+            self.speed[1] = -self.speed[1]
+
+        self.rect.left = self.walls(self.rect.left, 0 , screen_width)
+        self.rect.right = self.walls(self.rect.right, 0 , screen_width)
+        self.rect.top = self.walls(self.rect.top, 100 , screen_height)
+        self.rect.bottom = self.walls(self.rect.bottom, 0 , screen_height)
+
+        self.speed[1] += gravity                                        # simulates effect of gravity
+
+    def walls(self,number, min_number, max_number):
+        """ helper function for ball to bounce off of walls """
+        self.number = number
+        self.min_number = min_number
+        self.max_number = max_number
+        return min(max(self.number,self.min_number),self.max_number)
+
+class GameModel(object):
+    """ Model for game """
+
+    def __init__(self):
+        self.width = screen_width
+        self.height = screen_height
+
+        self.count = 1
+        self.score = 0
+        self.lives = 3
+        self.player1 = Player()
+
+        self.alive = self.player1.alive
+
+        self.balls = []
+
+        self.character_hit_list = []
+        self.bubble_hit_list = []
+
+        self.more_balls()
+
+        self.list_of_bubbles = pygame.sprite.Group()
+        self.player_list = pygame.sprite.Group()
+        self.gun_list = pygame.sprite.Group()
+
+    def more_balls(self):
+        """ Generates/adds more balls once the player has cleared all other balls """
+        # self.original_size = 4
+        # self.original_x_speed = 5
+        # self.original_y_speed = 3
```

Comment:
It's good practice to clean up comments and unused code, though it seems like you were thinking of passing in these values as arguments instead of hard-coding them?